javaslang/src/test/java/javaslang/collection/euler/Euler42Test.java

```diff
@@ -0,0 +1,87 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2016 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.collection.euler;
+
+import javaslang.Function1;
+import javaslang.collection.CharSeq;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+import static javaslang.collection.euler.Utils.file;
+import static javaslang.collection.euler.Utils.readLines;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Euler42Test {
+
+    /**
+     * <strong>Problem 42 Coded triangle numbers</strong>
+     * <p>
+     * The <i>n</i><sup>th</sup> term of the sequence of triangle numbers is
+     * given by, <i>t</i><sub>n</sub> = ½<i>n</i>(<i>n</i>+1); so the first ten
+     * triangle numbers are:
+     * <pre>
+     * 1, 3, 6, 10, 15, 21, 28, 36, 45, 55, ...
+     * </pre>
+     * <p>
+     * By converting each letter in a word to a number corresponding to its
+     * alphabetical position and adding these values we form a word value. For
+     * example, the word value for SKY is 19 + 11 + 25 = 55 = t<sub>10</sub>. If
+     * the word value is a triangle number then we shall call the word a
+     * triangle word.
+     * <p>
+     * Using words.txt (right click and 'Save Link/Target As...'), a 16K text
```

Comment:
Please rewrite this line and name p042_words.txt instead, we can't right-click it here.